ltm/agent.py

```diff
@@ -64,7 +64,12 @@ class LTMAgent:
 
     @property
     def save_name(self) -> str:
-        return f"{self.model}-{self.max_prompt_size}-{self.max_completion_tokens}"
+        model = self.model.replace("/", "-")
+        return f"{model}-{self.max_prompt_size}-{self.max_completion_tokens}"
+
+    @property
+    def max_input_tokens(self) -> int:
+        return int(0.9 * (self.max_prompt_size - self.max_completion_tokens))
```

Comment:
We should pick between `max_prompt_size` and `max_input_tokens` and use it consistently. That, or properly draw a distinction between them. Like `max_prompt_size` is how many tokens the model can deal with, and `max_input_tokens = max_prompt_size - max_response_tokens`

Reply:
Yeah, they are not the same, and that is exactly the difference.

Reply:
Oh yeah, I completely glossed over the fact that I included the definition of `max_input_tokens` in my comment :P